docs/getting-started-guides/coreos/azure/index.md

```diff
@@ -1,246 +0,0 @@
----
----
-
-* TOC
-{:toc}
-
-
-In this guide I will demonstrate how to deploy a Kubernetes cluster to Azure cloud. You will be using CoreOS with Weave, which implements simple and secure networking, in a transparent, yet robust way. The purpose of this guide is to provide an out-of-the-box implementation that can ultimately be taken into production with little change. It will demonstrate how to provision a dedicated Kubernetes master and etcd nodes, and show how to scale the cluster with ease.
-
-### Prerequisites
-
-1. You need an Azure account.
-
-## Let's go!
-
-To get started, you need to checkout the code:
-
-```shell
-https://github.com/weaveworks-guides/weave-kubernetes-coreos-azure
-cd weave-kubernetes-coreos-azure
-```
-
-You will need to have [Node.js installed](http://nodejs.org/download/) on you machine. If you have previously used Azure CLI, you should have it already.
-
-First, you need to install some of the dependencies with
-
-```shell
-npm install
-```
-
-Now, all you need to do is:
-
-```shell
-./azure-login.js -u <your_username>
-./create-kubernetes-cluster.js
-```
-
-This script will provision a cluster suitable for production use, where there is a ring of 3 dedicated etcd nodes: 1 kubernetes master and 2 kubernetes nodes. The `kube-00` VM will be the master, your work loads are only to be deployed on the nodes, `kube-01` and `kube-02`. Initially, all VMs are single-core, to ensure a user of the free tier can reproduce it without paying extra. I will show how to add more bigger VMs later.
-If you need to pass Azure specific options for the creation script you can do this via additional environment variables e.g.
-
-```shell
-AZ_SUBSCRIPTION=<id> AZ_LOCATION="East US" ./create-kubernetes-cluster.js
-# or
-AZ_VM_COREOS_CHANNEL=beta ./create-kubernetes-cluster.js
-```
-
-![VMs in Azure](/images/docs/initial_cluster.png)
```

Comment:
Once the file in the root directory gets removed (per @errordeveloper ), I can merge this for you.